src/web/content/static/css/letterpress.css

```diff
@@ -367,17 +367,17 @@ a.sms-large-font:hover {
 
 /* Home ******************* */
 .sms-home-main-container {
-    height: 80vh;
+    height: 85vh;
     display: flex;
     flex-direction: column;
 }
 .sms-home-header {
-    height: 20vh;
+    height: 15vh;
 }
 
 .sms-home-intro-txt {
-    margin-top: 5.875rem;
```

Comment:
This was some other sloppy spacing issue I missed for the "Search real content by real people from..." copy.